.vitepress/theme/components/PreferenceSwitch.vue

```diff
@@ -168,7 +168,13 @@ function useToggleFn(
 
 .switch-container {
   display: flex;
-  align-items: center;
+  align-items: center;  
+}
+
+@media(max-width:959px){
+  .switch-container {
+    padding:0 1em;
```

Comment:
This is still missing the spaces after the colons:

```suggestion
@media(max-width: 959px){
  .switch-container {
    padding: 0 1em;
```